guide/src/whirlwind-tour/basic-usage.md

```diff
@@ -170,6 +174,14 @@ $ npm run serve
 If you open [http://localhost:8080](http://localhost:8080) in a browser you should see a `Hello, world!`
 dialog pop up!
 
+To publish the application, simply run:
+
+```shell
+$ npx webpack
```

Comment:
Well we can just use npm run webpack. Simpler and default

Reply:
That doesn't work: `npm ERR! missing script: webpack`. I think `npm run` only works with scripts defined in your `package.json`, like `serve` is defined in the guide.

I'm not sure what you mean by default, `npx` seems to be part of `npm`: https://medium.com/@maybekatz/introducing-npx-an-npm-package-runner-55f7d4bd282b.

Reply:
Oops, Well I was under the assumption that we already have the script. 

By default, I mean everybody uses that and knows about it.

Reply:
Well, `npx` sounds easier, since that's what it's intended for (running tools from `.bin`), and there's no need to add something to `package.json`.